contracts/src/Staking.sol

```diff
@@ -15,33 +15,43 @@ contract Staking is Ownable {
     // ============================================================
 
     /*
```

Comment:
Note that this isn't valid NatSpec - it _MUST_ be either `/** */` or `///`. I personally prefer the latter (its much easier to properly indent and you don't need to manage closing tags), but we should fix all the NatSpec in the contract.